compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1651,6 +1655,18 @@ int64_t CustomOp::getRank(Value v) {
   return cast<RankedTensorType>(type).getRank();
 }
 
+unsigned CustomOp::getNumNonLoopDimensions() {
+  for (auto map : llvm::map_range(getIndexingMaps(), [](Attribute attr) {
```

Comment:
nit: Add a getIndexingMapArray method that does this map_range to CustomOp and use that.

Reply:
I dont want to create an array to just iterate over. This just reads in place.

Reply:
I think you can write `for (auto map : getIndexingMaps().getAsValueRange<AffineMapAttr>())`, which avoids a lamda.